.scalafix.conf

```diff
@@ -1,4 +1,4 @@
-rules = [DisableSyntax, NoValInForComprehension, ProcedureSyntax, OrganizeImports]
+rules = [DisableSyntax, NoValInForComprehension, OrganizeImports]
```

Comment:
sbt-projectmatrix added the impact of having scalafmt/scalafix operations run for each project inside of the matrix, including scala 3. These updates allow us to run it on scala 3.